lib/events/dynamoevents/dynamoevents.go

```diff
@@ -732,6 +733,20 @@ func (notReadyYetError) Error() string {
 	return "The DynamoDB event backend is not ready to accept queries yet. Please retry in a couple of seconds."
 }
 
+func eventFilterList(amount int) string {
+	list := "("
+
+	if amount != 0 {
+		list += ":eventType0"
```

Comment:
hm, could you elaborate what "eventType0" and eventType" + n  mean ?

Reply:
These are just autogenerated variable names that we use for the types.